dl4j-distributed-training-examples/src/main/java/org/deeplearning4j/distributedtrainingexamples/patent/DownloadPreprocessPatents.java

```diff
@@ -99,7 +99,7 @@
     private int maxSequenceLength = 1000;
 
     @Parameter(names = {"--wordVectorsPath"})
-    private String wordVectorsPath = "wasbs://resources@dl4jdata.blob.core.windows.net/wordvectors/GoogleNews-vectors-negative300.bin.gz";
+    private String wordVectorsPath = "wasbs://resources@deeplearning4jblob.blob.core.windows.net/wordvectors/GoogleNews-vectors-negative300.bin.gz";
```

Comment:
This needs to be changed to "wasbs://resources@dl4jdata.blob.core.windows.net/wordvectors/GoogleNews-vectors-negative300.bin.gz";

Reply:
Done